src/lib.rs

```diff
@@ -207,6 +237,11 @@ pub trait BitLength {
 }
 
 pub trait Hashable {
+    /// # Message Digest
+    /// Computes SHA3-d hash of input.
+    /// ## Arguments:
+    /// * `d: u64`: requested security strength in bits. Supported
+    /// bitstrengths are 224, 256, 384, or 512.
```

Comment:
because this is simply a trait definition, its never really publicly accessed aside from its implementations, which appear in ```ops.rs```, so no docs are needed on this